crux-bench/bin/download_dataset.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+cd crux-bench/
```

Comment:
you probably want `cd $(dirname "$0")/..` so that this works even if it's not run from the `crux-bench` dir, and a `cd -` at the end of the script to go back to the user's previous directory

Reply:
Quickly added that so that circle could use it, seems good to change it :+1: